graphql_api/tests/test_repository.py

```diff
@@ -885,6 +885,7 @@ def test_fetch_is_github_rate_limited_but_errors(
             },
         )
 
+    # TODO - remove this with #2291
```

Comment:
These tests that are marked TODO remove are duplicated (with minor adjustments to work with new level of nesting) at new `test_test_analytics.py`